src/main/java/org/gradlex/buildparameters/BuildParametersExtension.java

```diff
@@ -35,6 +37,7 @@ public BuildParametersExtension(GradlePluginDevelopmentExtension gradlePlugins)
             p.setId("build-parameters");
             p.setImplementationClass(PACKAGE_NAME + "." + PLUGIN_CLASS_NAME);
         });
+        getDisableValidation().convention(false);
```

Comment:
⭕️ Let's invert this property to be called `enabledValidation` with default `true`